packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -376,13 +376,19 @@ export class DataStores implements IDisposable {
     public resubmitDataStoreOp(content: any, localOpMetadata: unknown) {
         const envelope = content as IEnvelope;
         const context = this.contexts.get(envelope.address);
+        // The implicit coercion below also informs TypeScript that context cannot be null. Removing the implicit
+        // coercion here would make the subsequent code more complex.
+        // eslint-disable-next-line no-implicit-coercion
         assert(!!context, 0x160 /* "There should be a store context for the op" */);
```

Comment:
Most of these `!!thing` usages should really be `thing !== undefined`, which is a stronger check anyway.

Reply:
Ohhhh, that's a way better fix... I'll update where I can. I hate having these comments everywhere.